src/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
+<html lang="en" class="page">
+  <head >
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        initial-scale=1.0
+      ">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <!-- Is also an element of page so we can apply margin -->
+    <div class="calendar calendar--month_length-29 calendar--start_day-fri page__calendar">
+      <div class="calendar__day-1"></div>
+      <div class="calendar__day-2"></div>
+      <div class="calendar__day-3"></div>
+      <div class="calendar__day-4"></div>
+      <div class="calendar__day-5"></div>
+      <div class="calendar__day-6"></div>
```

Comment:
think of a way of making loop that helps you to avoid writing `...day-x` explicitly
You don't want anything to be hardcoded in your code

Reply:
That's true , ill have to think more next time!